analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -140,6 +140,36 @@ class MavenSupportTest : WordSpec({
                 path = "boringssl-static"
             )
         }
+
+        "handle GitHub URLs with double 'git:' prefix" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git:git:github.com/MarkusAmshove/Kluent.git"
```

Comment:
Do you think the metadata intended but forgot to specify the protocol, so intention was to specify the format as in line 132? If so, should we add `git` as a protocol to the result?